packages/protocol/migrations_sol/Migration.s.sol

```diff
@@ -198,6 +209,8 @@ contract Migration is Script, UsingRegistry {
 
     migrateFreezer();
     migrateFeeCurrencyWhitelist();
+    // TODO(Arthur): Add FeeCurrencyDirectory here
```

Comment:
Do we need this todo ?

Reply:
Can you please check whether all todos are needed ?

Reply:
No, I removed various TODO comments I left for myself in this commit https://github.com/celo-org/celo-monorepo/commit/18eb1638618457022c443c6571968eb47dbe9848

Reply:
> Do we need this todo ?

No, I removed various TODO comments I left for myself in this commit https://github.com/celo-org/celo-monorepo/commit/18eb1638618457022c443c6571968eb47dbe9848

Reply:
> Can you please check whether all todos are needed ?

Yes, that's my bad, the PR is not ready for review. I need to remove a couple `TODO` comments I left for myself, and add a PR description.